### PR TITLE
[Synfig Studio] Align close button of animation tabs to the right for better looking layout

### LINKED ISSUE #2127: Wide spaces between close button and tab's edge when multiple tabs are opened
**Synfig version & platform**:
Building from source
Fedora 33

**Issue description**:
Is this an expected behavior?
https://user-images.githubusercontent.com/24214509/116174354-dbeb5f00-a72b-11eb-821f-a4c74fec81c2.mov

### PR DESCRIPTION
**Pull request by FirasH**
Fix #2127

### CODE REVIEW

File: synfig-studio/src/gui/docks/dockbook.cpp [hunk 1]
```diff
@@ -168,6 +168,7 @@ DockBook::refresh_tab(Dockable* dockable)
 	);
 
 	set_tab_label(*dockable, *label);
+	label->set_halign(Gtk::ALIGN_CENTER);
```
Comment (rodolforg) [inline synfig-studio/src/gui/docks/dockbook.cpp:171]: This method affects all tabs, right? Not only canvas views.

Comment (FirasH) [reply synfig-studio/src/gui/docks/dockbook.cpp:171]: Yes, that's right

Comment (FirasH) [reply synfig-studio/src/gui/docks/dockbook.cpp]: With the last commit only document tabs are affected.

File: synfig-studio/src/gui/canvasview.cpp [hunk 2]
```diff
@@ -1980,7 +1983,6 @@ CanvasView::create_tab_label()
 	label->show();
 
 	closebutton = manage(new Gtk::Button());
-	closebutton->set_margin_end(4);
```
Comment (Keyikedalube) [inline synfig-studio/src/gui/canvasview.cpp:1983]: On GNOME desktop default Adwaita theme, close button is hidden slightly: https://github.com/synfig/synfig/pull/1774#discussion_r504351483

That line is not safe to be deleted. Maybe you could add a single-line comment above so other contributors know why.

**Review by Keyikedalube — CHANGES_REQUESTED** [review]

File: synfig-studio/src/gui/canvasview.cpp (hunk 2 above)
Comment (FirasH) [reply synfig-studio/src/gui/canvasview.cpp:1983]: Thanks, didn’t know that 👍

**Review by Keyikedalube — APPROVED** [review]

**Comment by morevnaproject** [comment]
I suggest to change title of this PR. ^__^

**Comment by rodolforg** [comment]
@FirasH  please squash and force push :)

**Comment by FirasH** [comment]
> @FirasH please squash and force push :)

Done :+1:

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**